drivers/SmartThings/zigbee-switch/src/zigbee-dual-metering-switch/init.lua

```diff
@@ -25,31 +36,61 @@ local function can_handle_zigbee_dual_metering_switch(opts, driver, device, ...)
   return false
 end
 
-local function endpoint_to_component(device, endpoint)
-  if endpoint == 2 then
-    return "switch1"
-  else
-    return "main"
+local function do_refresh(self, device)
+  for endpoint = ENDPOINTS.parent, ENDPOINTS.child do
+    device:send(OnOff.attributes.OnOff:read(device):to_endpoint(endpoint))
+    device:send(SimpleMetering.attributes.Divisor:read(device):to_endpoint(endpoint))
+    device:send(SimpleMetering.attributes.Multiplier:read(device):to_endpoint(endpoint))
+    device:send(ElectricalMeasurement.attributes.ACPowerDivisor:read(device):to_endpoint(endpoint))
+    device:send(ElectricalMeasurement.attributes.ACPowerMultiplier:read(device):to_endpoint(endpoint))
```

Comment:
@NikitaKuchta I think we may refresh only one device at a time. 
Could You also write unit tests for that ?

Reply:
done